safe-global/safe-wallet-web
# fix: Show remaining countdown if delay is less than a minute

## What it solves

Resolves [No visible countdown if < 60 seconds](https://www.notion.so/safe-global/af7bf82ec5a040d3acc20b167bb0656a?v=27045d3b2d0a4c938f627bb5f23bc7f5&p=8f5f63f31207419aa45e255e75aa1553&pm=s)

## How this PR fixes it

- Shows a message `< 1 min` if less than a minute delay remaining 
- Uses the `formatDateTime` for the recovery tx in case the expiry is more than a day out
- Fixes lint issues with renaming a `transactionHash` to `recoveryTxHash`

## How to test it

1. Open a safe with recovery enabled
2. Create a recovery proposal tx
3. Observe the dashboard and queue and modal saying "< 1 min" once that delay time is reached

## Screenshots

<img width="393" alt="Screenshot 2023-11-29 at 15 46 51" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/5947e4b3-cd7f-42ea-9a78-70f3419db499">



## Checklist
